apps/package.json

```diff
@@ -59,9 +60,11 @@
     "grunt-lodash": "0.3.0",
     "grunt-newer": "1.1.1",
     "grunt-notify": "0.4.1",
+    "grunt-open": "^0.2.3",
     "grunt-sass": "1.0.0",
     "grunt-strip-code": "0.1.2",
     "http-proxy": "1.11.1",
+    "jit-grunt": "^0.9.1",
```

Comment:
Currently un-used in the gruntfile. It sped the grunt dev build time up .5-1.5s, but messed with the grunt timing output a bit. Want to do a bit more experimentation before pushing. I'll remove from gruntfile